### DISCUSSION TITLE
Lost access to my account....

### DISCUSSION THREAD

**Discussion by Mycrypto7** [community/community#34514] [Discussions]
Had to create new account to speak in here....Here si my troubelshhot:
I changed my email in my account...Now I lost access to this mailto and even if I know username & password system asks me to verify trough a email code they send to me, trouble is I cannot access my email, I'm log out of my main account...I have a pgp key installed in my main account but 2fa is off maybee it can help?

Someone can maybee give me a little help to recover strange I''m using same ip address than I use to suddenly this verification pop up.
Thanks foir your nice help

**Comment by radar-blip** [comment]
For account related issues, see here: [Support page](https://support.github.com/contact/account)
